Cloud Application Manager/API/centurylink-private-node.md

```diff
@@ -0,0 +1,54 @@
+{{{ "title": "CenturyLink Private Node",
```

Comment:
@jtowns3 i've replaced your request with https://github.com/CenturyLinkCloud/PublicKB/pull/1729 which is a new kb category, icon & included your code in it.  closing this request since that is now covered in the new pull